### PR TITLE
Feature/revert breaking change

### PR DESCRIPTION
**Pull request by HenriBeck**
## Corresponding issue (if exists):

#1249 

## Todo

- [ ] Add tests if possible
- [x] Add changelog if users should know about the change
- [ ] Add documentation

### CODE REVIEW

**Comment by kof** [comment]
What did it break?

**Comment by HenriBeck** [comment]
> What did it break?

#1249

**Comment by kof** [comment]
oh shit, `return this` in the method is what I broke there.

**Comment by kof** [comment]
lets do the same for updateOne

**Comment by HenriBeck** [comment]
> lets do the same for updateOne

done

**Comment by HenriBeck** [comment]
@kof one last look?

**Comment by kof** [comment]
released

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Since you are interested in what happens next, in case, you work for a for-profit company that benefits from using the project, please consider supporting it on https://opencollective.com/jss.
 
+### Bug fixes
+
+- [jss] Fix breaking change that was introduced in `10.0.1` ([1252](https://github.com/cssinjs/jss/pull/1252))
+
 ---
```
Comment (kof) [inline changelog.md]: uhh, that thing was supposed to be above the changelog entries, that's why release message was broken, we should validate this somehow or make it more obvious

**Pull request merged**